contents/handbook/strategy/overview.md

```diff
@@ -8,6 +8,8 @@ showTitle: true
 
 Our mission is to increase the number of successful products in the world.
 
+We do this by building the best developer tool for **engineers** to build better products.
```

Comment:
```suggestion
We do this by helping **engineers** become better at product.
```

I think this better captures what we're doing - ie the description above could mean _any _ dev tool - CI/CD / a programming language / GitHub, which I don't think is in scope

I debated saying "product, marketing and sales", but I think that's maybe too broad?